hokusai/development.yml

```diff
@@ -2,9 +2,9 @@
 version: "2"
 services:
   horizon:
+    env_file: ../.env
     environment:
       - RAILS_ENV=development
-      - RAILS_LOG_TO_STDOUT=true
```

Comment:
Why remove this? (Do the defaults change in rails 6? If so the code that references it in the environment files can be removed.)

Reply:
We reference this in the `build.yml` so it shouldn't need to go into individual environments.